### PR TITLE
Discover data providers from other classes

### PR DESCRIPTION
**Pull request by villfa**
Fix for the issue highlighted by #153.

### CODE REVIEW

File: src/Rules/PHPUnit/DataProviderHelper.php [hunk 1]
```diff
@@ -5,16 +5,32 @@
...
+	/**
+	 * Reflection provider.
+	 *
+	 * @var ReflectionProvider
+	 */
+	private $reflectionProvider;
+
+	public function __construct(ReflectionProvider $reflectionProvider)
+	{
+		$this->reflectionProvider = $reflectionProvider;
```
Comment (kubawerlos) [inline src/Rules/PHPUnit/DataProviderHelper.php:31]: So there is auto wiring it the project? I wanted to fix it by myself, but failed on getting reflection of other class and it is that simple :)

**Review by kubawerlos — APPROVED** [review]
👍🏼

File: src/Rules/PHPUnit/DataProviderHelper.php (hunk 1 above)
Comment (ondrejmirtes) [reply src/Rules/PHPUnit/DataProviderHelper.php:31]: https://phpstan.org/developing-extensions/dependency-injection-configuration#services

**Comment by ondrejmirtes** [comment]
Thank you. I can release this after PHPStan 1.9.3 is released.

**Pull request merged**